src/Microsoft.SqlTools.SqlCore/ObjectExplorer/ObjectExplorer.cs

```diff
@@ -0,0 +1,221 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+using System;
+using System.Collections.Concurrent;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Microsoft.Data.SqlClient;
+using Microsoft.SqlServer.Management.Common;
+using Microsoft.SqlTools.SqlCore.Connection;
+using Microsoft.SqlTools.SqlCore.ObjectExplorer.Nodes;
+using Microsoft.SqlTools.SqlCore.ObjectExplorer.SmoModel;
+using Microsoft.SqlTools.Utility;
+
+namespace Microsoft.SqlTools.SqlCore.ObjectExplorer
+{
+    public class SimpleObjectExplorer
```

Comment:
Please add XML doc explaining what this is and how it differs from the "other" object explorer class

(class names should also generally match the filename. So would suggest updating one or the other)

Reply:
Removing this class for now.